src/main/java/com/artipie/RepoConfig.java

```diff
@@ -199,8 +199,10 @@ public Storage storage() {
      * @return Async YAML mapping
      */
     public YamlMapping repoConfig() {
-        return Optional.ofNullable(this.yaml.yamlMapping("repo")).orElseThrow(
-            () -> new IllegalStateException("Invalid repo configuration")
+        return Optional.ofNullable(this.yaml.yamlMapping("repo")).orElse(
+            Yaml.createYamlMappingBuilder()
```

Comment:
I don't think it's a good solution to implicitly create configuration for repo in `src/main` - it may cause unpredictable behavior for users. I'd be better to set up proper configuration in for test.

Reply:
Ok

Reply:
@g4s8 I think it's normal to respond `500` when we haven't a valid repo config. So `startsWhenNotValidRepoConfigsArePresent` should rather expect `500` instead of `200`. WDYT ?